index.html

```diff
@@ -50,35 +50,79 @@ <h1>My Name</h1>
     <main tabindex="0">
       <section id="about">
         <header><h2>About Me</h2></header>
-        <p>
-          Lorem, ipsum dolor sit amet consectetur adipisicing elit. Nulla nisi
-          excepturi quidem, eum rem doloribus quam aspernatur hic enim eligendi
-          commodi minima ullam necessitatibus, cumque blanditiis, nihil magni
-          amet consectetur?
+        <p class="abt-me">
+          I'm Berwin Eibo, a dedicated software developer with a background in
+          education. My years of experience in teaching have honed my
+          communication skills and taught me the value of teamwork. I'm
+          passionate about creating innovative solutions to real-world problems
+          through code. I thrive on learning and eagerly embrace new challenges
```

Comment:
Great start - I like that you bring in your experience and your passion. Run this through ResumeWorded to reduce the use of buzzword.